components/developer/byacc/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2025 David Stes
+#
+
+BUILD_BITS= 64
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		byacc
+COMPONENT_VERSION=	20241231
+COMPONENT_SUMMARY=	byacc - Berkeley Yacc
+COMPONENT_DESCRIPTION=  Berkeley Yacc is an LALR(1) parser generator
+COMPONENT_PROJECT_URL=	https://invisible-island.net/archives/byacc
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tgz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:192c2fae048d4e7f514ba451627f9c4e612765099f819c19191f9fde3e609673
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=		developer/parser/byacc
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+# Berkeley Yacc is reportedly not BSD software but PublicDomain (see LICENSE file in the package)
+COMPONENT_LICENSE=	PublicDomain
+
```

Comment:
Please add
COMPONENT_LICENSE_FILE= LICENSE
as the sources already have it and remove the explicit license file you added to the PR.